packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -265,12 +264,4 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
 
 		return url.href;
 	}
-
-	/**
-	 * Crafts a supported data store nav param
-	 * @deprecated encodeOdspFluidDataStoreLocator should be used instead
-	 */
-	public static createNavParam(locator: OdspFluidDataStoreLocator) {
-		return encodeOdspFluidDataStoreLocator(locator);
-	}
```

Comment:
I think you'll probably need to generate a changeset as this is a breaking change. https://github.com/microsoft/FluidFramework/wiki/Changesets